- bioimageio_core_version: 0.5.8
  bioimageio_spec_version: 0.4.9
  error: null
  name: load resource description
  nested_errors: null
  source_name: scripts/../bioimageio-gh-pages/rdfs/10.5281/zenodo.5910854/6539073/rdf.yaml
  status: passed
  traceback: null
  warnings: {}
- {error: null, name: has expected resource type, source_name: 10.5281/zenodo.5910854/6539073,
  status: passed, traceback: null}
- bioimageio_core_version: 0.5.8
  bioimageio_spec_version: 0.4.9
  error: null
  name: All URLs and paths available
  nested_errors: null
  source_name: 10.5281/zenodo.5910854/6539073
  status: passed
  traceback: null
  warnings: {}
- bioimageio_core_version: 0.5.8
  bioimageio_spec_version: 0.4.9
  error: null
  name: Test documentation completeness.
  source_name: 10.5281/zenodo.5910854/6539073
  status: passed
  traceback: null
  warnings: {documentation: No '# Validation' (sub)section found.}
- bioimageio_core_version: 0.5.8
  bioimageio_spec_version: 0.4.9
  error: No module named 'onnxruntime'
  name: reproduce test outputs from test inputs
  source_name: 10.5281/zenodo.5910854/6539073
  status: failed
  traceback: ["  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
      line 157, in _test_model_inference\n    bioimageio_model=model, devices=devices,
      weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
      line 205, in create_prediction_pipeline\n    bioimageio_model=bioimageio_model,
      devices=devices, weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
      line 132, in create_model_adapter\n    adapter_cls = _get_model_adapter(weight)\n",
    "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
      line 157, in _get_model_adapter\n    from ._onnx_model_adapter import ONNXModelAdapter\n",
    "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_onnx_model_adapter.py\",
      line 5, in <module>\n    import onnxruntime as rt\n"]
  warnings: {}
